### DISCUSSION TITLE
Can't follow the last 2 steps of the instructions.

### DISCUSSION THREAD

**Discussion by Sentry03** [jbergler/hass-ttlock#99] [Q&A] [answered]
Once the integration is working you should receive a system notification with the webhook url
This will go away when the webhook receives data
Go back to the url from the first step and set the 'Callback URL' for your application

I don't know where to look for this system notification with the webhook url.
also what should I put on the callback URL?

![image](https://github.com/jbergler/hass-ttlock/assets/155286392/c52af93b-1611-4363-933b-7678ae5c69a3)
I'm hoping to get notifications or to see the state of the lock.

**Comment by jbergler** [comment, marked as answer]
The notifications should be in the bottom left of the browser UI, just above the user profile link.

![Screenshot from 2024-02-20 09-17-49](https://github.com/jbergler/hass-ttlock/assets/698597/8199e676-2e02-4cd3-afb5-134efcfe8892)

**Reply by Sentry03** [reply]
thank you very much. Got it working now, felt so stupid that I haven't been paying attention to that tab lol

**Comment by nigels0** [comment]
I'm having a similar issue; I've connected the developer account and when installing the integration, it all works fine and I get a web hook URL. I put that into the developer account page on the callback URL - but I can't see how it could work as it is a local IP address.

**Reply by nmartyd** [reply]
did you figure it out?  Do I need a public IP and to open ports in my router?

**Reply by nigels0** [reply]
No. I didn’t need to expose the router, but I get an invalid login every time I restart HA
